frontend/src/pages/LandingPage/LandingPage.js

```diff
@@ -1,18 +1,35 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { 
-    Box, Typography, Container
+    Box, Container
   } from '@mui/material'
 import Retool from 'react-retool'
 
-const LandingPage = () => { 
+const LandingPage = ({
+  externalIdentifier = "",
+  groups = [5,6],
+  pageUuid = "abc"
+}) => { 
+
+  const [retoolEmbedUrl, setRetoolEmbedUrl] = useState([])
+
+  useEffect(() => {
+    const options = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ externalIdentifier, groups, pageUuid })
+    };
+    
+    fetch('/embedUrl', options)
+    .then((res) => res.json())
+    .then(data => setRetoolEmbedUrl(data.embedUrl))
+  })
```

Comment:
This LGTM from a React flow point of view, but per my other comment the frontend should not determine `externalIdentifier`, `groups`, or `pageUuid`